pkg/controller/mizar/mizar-arktos-network-controller.go

```diff
@@ -176,6 +195,36 @@ func (c *MizarArktosNetworkController) syncNetwork(eventKeyWithType KeyWithEvent
 func (c *MizarArktosNetworkController) processNetworkCreation(network *v1.Network, eventKeyWithType KeyWithEventType) error {
 	//skip update or create if type is not mizar or network status is ready
 	key := eventKeyWithType.Key
+
+	if network.Spec.Type == mizarNetworkType && network.Status.Phase == v1.NetworkReady {
+		const subnetSuffix = "-subnet"
+		vpc := network.Spec.VPCID
+		subnet := vpc + subnetSuffix
+
+		const homeSubPath = "/go/src/arktos/hack/runtime/"
+		templateDir := "/tmp/runtime/"
```

Comment:
similar here, what if the host is rebooted and the /tmp/ folder is auto cleaned ?

Reply:
If $HOME is not set, need manually create two template files under /tmp/runtime. If $HOME is set, the template files are ./hack/runtime.

Reply:
Find the method and is testing in poc branch.

Reply:
new codes are tested successfully in poc branch.